pretixprint/app/src/main/java/eu/pretix/pretixprint/print/ESCPOSRenderer.kt

```diff
@@ -634,4 +668,30 @@ class ESCPOSRenderer(private val receipt: JSONObject, private val charsPerLine:
         out.add(durationOn.toByte())
         out.add(durationOff.toByte())
     }
+
+    fun renderTestPage(): ByteArray {
+        out.clear()
+        init()
+        if (dialect == Dialect.EpsonDefault) {
+            characterCodeTable(CharacterCodeTable.WPC1252.codeTable)
```

Comment:
Testpage character code table `WPC1252` doesn't match the one used in normal prints (line 106: `UserDefined2`)

Reply:
Ooops, the change in render was unintentional, thanks